website/content/docs/commands/license/index.mdx

```diff
@@ -7,12 +7,10 @@ description: |
 
 # Command: license
 
-~> License commands are new in Nomad 0.12.0 and are only available with Nomad Enterprise.
+~> License commands are only available with Nomad Enterprise.
 
-The `license` command provides cluster level management of the Nomad Enterprise license.
-
-If ACLs are enabled then a token with operator privileges may be required in order to use
-this command.
```

Comment:
If we are removing this, do we want to document this in other places? I feels like a nice thing to have ACL details easy to see for operators.

Reply:
It's still located where it's needed in the `license get` doc: https://developer.hashicorp.com/nomad/docs/v1.5.x/commands/license/get#usage -- think that's enough?

Reply:
oh, and @lgfa29 re:

> I believe the ACL requirement was removed in https://github.com/hashicorp/nomad/pull/10458. The remaining commands don't make any HTTP or RPC requests, so there are no ACL requirements for them anymore.

`license get` does still get the license (metadata) from the server via API call.  I only removed the ACL note from this index page because it doesn't apply to both commands.